stable/weave-cloud/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+Verify all Pods are running
```

Comment:
Thanks for the pointer! I think `required` function seems more suitable, I think `NOTES.txt` solution predates `required`...